integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -390,10 +383,17 @@ def export(self, **kwargs):
             content_keys = self._get_catalog_content_keys(enterprise_customer_catalog)
 
             self._log_info(
-                f'Retrieved content keys: {content_keys} for past transmissions to customer: '
+                f'Retrieved {len(content_keys)} content keys for past transmissions to customer: '
```

Comment:
logging count here, then individual keys on their own line after. logging all the keys in-line like this just led to truncated log lines in splunk, so I cant tell if the course i am looking for is making it to this step.